src/main/java/org/jitsi/videobridge/util/PartitionedByteBufferPool.java

```diff
@@ -44,12 +44,7 @@
     /**
      * How many buffers to pre-allocate in each partition.
      */
-    private static final int INITIAL_SIZE = 100;
-
-    /**
-     * The size of buffers to allocate.
-     */
-    private static final int DEFAULT_BUFFER_SIZE = 1500 + RtpPacket.BYTES_TO_LEAVE_AT_END_OF_PACKET;
+    private static final int INITIAL_SIZE = 10;
```

Comment:
10 seems really low?

Reply:
We now have 24 partitions in total. Allocating 2400 initial buffers seemed excessive. In fact, I don't think pre-allocating achieves anything